src/main/java/blackjack/domain/gamer/Gamer.java

```diff
@@ -0,0 +1,63 @@
+package blackjack.domain.gamer;
+
+import blackjack.domain.card.Card;
+import blackjack.domain.card.Cards;
+import java.util.List;
+
+public abstract class Gamer {
+	protected final Cards cards;
+	private final Name name;
+
+	protected Gamer(Cards cards, Name name) {
+		this.cards = cards;
+		this.name = name;
+	}
+
+	public void addCards(List<Card> cards) {
+		cards.forEach(this.cards::addCard);
+	}
+
+	public abstract boolean canHit();
+
+	public abstract boolean isWin(Gamer gamer);
+
+	public boolean isDraw(Gamer gamer) {
+		return !isWin(gamer) && !gamer.isWin(this);
+	}
+
+	public boolean isLose(Gamer gamer) {
+		return !isWin(gamer) && !isDraw(gamer);
+	}
+
+	protected boolean isBlackJackWin(Gamer gamer) {
+		return this.isBlackJack() && !gamer.isBlackJack();
+	}
+
+	protected boolean isHigherScore(Gamer gamer) {
+		return this.getScore() > gamer.getScore();
+	}
+
+	protected boolean isNotBustBoth(Gamer gamer) {
+		return !this.isBust() && !gamer.isBust();
+	}
+
+	public List<Card> getCards() {
+		return this.cards.getCards();
+	}
+
+	public int getScore() {
+		return this.cards.getScore();
+	}
+
+	public String getName() {
+		return this.name.getName();
+	}
+
+	public boolean isBust() {
+		return this.cards.isBust();
+	}
+
+	public boolean isBlackJack() {
+		return this.cards.isBlackJack();
+	}
```

Comment:
컨벤션상 getter는 일반 메서드보다 아래에 위치하도록 합니다.
기본 들여쓰기도 4 tab이 아니라 4 space입니다!
우아한테크코스 자바 스타일 가이드 적용 후 전체 코드에 대해 반영 부탁드릴게요!
* [우아한테크코스의 자바 스타일 가이드](https://github.com/woowacourse/woowacourse-docs/tree/master/styleguide/java)

```suggestion
        public boolean isBust() {
		return this.cards.isBust();
	}

	public boolean isBlackJack() {
		return this.cards.isBlackJack();
	}
	
	public List<Card> getCards() {
		return this.cards.getCards();
	}

	public int getScore() {
		return this.cards.getScore();
	}

	public String getName() {
		return this.name.getName();
	}
```

Reply:
수정하였습니다